test_unstructured/partition/test_email.py

```diff
@@ -540,135 +650,29 @@ def test_partition_email_can_process_min_max_with_attachments(
             assert len(element.text) >= 6
 
 
-def test_partition_msg_raises_with_no_partitioner(
```

Comment:
Did these tests get removed or is that just the diff being weird?

Reply:
The diff being weird from my refactoring and grouping similar tests together in the second commit

Reply:
Gotcha, thanks!